include/CppUTest/UtestMacros.h

```diff
@@ -142,6 +142,14 @@
     UtestShell::getCurrent()->assertLongsEqual((long)0, (long)0, NULL, file, line); \
   } }
 
+#define CHECK_RELATION(first, relop, second)\
+ { SimpleString conditionString = "evaluates to ";\
+   conditionString += StringFrom(first); conditionString += " ";\
+   conditionString += #relop; conditionString += " ";\
+   conditionString += StringFrom(second);\
+   CHECK_TRUE_LOCATION((first) relop (second), "CHECK", #first " " #relop " " #second, conditionString.asCharString(), __FILE__, __LINE__)\
```

Comment:
The second parameter to CHECK_TRUE_LOCATION should be the same than the macro ("CHECK_RELATION").